popup.php

```diff
@@ -16,63 +16,61 @@
+
 	if ( FORCE_SSL_ADMIN ) {
-		$formurl = str_replace( "http:", "https:", $formurl );
+		$formurl = str_replace( 'http:', 'https:', $form_url );
 	}
 	?>
 
-	<form enctype="multipart/form-data" method="post" action="<?php echo $formurl; ?>">
-		<?php
-		#wp_nonce_field('enable-media-replace');
-		?>
-		<input type="hidden" name="ID" value="<?php echo (int) $_GET["attachment_id"]; ?>"/>
+	<form enctype="multipart/form-data" method="post" action="<?php echo esc_url( $form_url ); ?>">
+		<?php wp_nonce_field('media_replace_upload'); ?>
+		<input type="hidden" name="ID" value="<?php echo esc_attr( $attachment_id ); ?>"/>
 
 		<div id="message" class="updated fade">
-			<p><?php echo __( "NOTE: You are about to replace the media file", "enable-media-replace" ); ?>
-				"<?php echo $current_filename ?>
-				". <?php echo __( "There is no undo. Think about it!", "enable-media-replace" ); ?></p></div>
+			<p><?php esc_html_e( 'NOTE: You are about to replace the media file', 'enable-media-replace' ); ?>
+				"<?php echo esc_html( $current_filename ); ?>
+				". <?php esc_html_e( 'There is no undo. Think about it!', 'enable-media-replace' ); ?></p></div>
 
-		<p><?php echo __( "Choose a file to upload from your computer", "enable-media-replace" ); ?></p>
+		<p><?php esc_html_e( 'Choose a file to upload from your computer', 'enable-media-replace' ); ?></p>
 
 		<input type="file" name="userfile"/>
 
 		<?php do_action( 'emr_before_replace_type_options' ); ?>
 
 		<?php if ( apply_filters( 'emr_display_replace_type_options', true ) ) : ?>
-			<p><?php echo __( "Select media replacement type:", "enable-media-replace" ); ?></p>
+			<p><?php esc_html_e( 'Select media replacement type:', 'enable-media-replace' ); ?></p>
 
-			<label for="replace_type_1"><input CHECKED id="replace_type_1" type="radio" name="replace_type" value="replace"> <?php echo __( "Just replace the file", "enable-media-replace" ); ?>
+			<label for="replace_type_1"><input CHECKED id="replace_type_1" type="radio" name="replace_type" value="replace"> <?php esc_html_e( 'Just replace the file', 'enable-media-replace' ); ?>
 			</label>
-			<p class="howto"><?php echo __( "Note: This option requires you to upload a file of the same type (", "enable-media-replace" ); ?><?php echo $current_filetype; ?><?php echo __( ") as the one you are replacing. The name of the attachment will stay the same (", "enable-media-replace" ); ?><?php echo $current_filename; ?><?php echo __( ") no matter what the file you upload is called.", "enable-media-replace" ); ?></p>
+			<p class="howto"><?php esc_html_e( 'Note: This option requires you to upload a file of the same type (', 'enable-media-replace' ); ?><?php esc_html_e( $current_filetype ); ?><?php esc_html_e( ') as the one you are replacing. The name of the attachment will stay the same (', 'enable-media-replace' ); ?><?php echo esc_html( $current_filename ); ?><?php esc_html_e( ') no matter what the file you upload is called.', 'enable-media-replace' ); ?></p>
```

Comment:
This should probably have been done with `sprintf`! No need to fix immediately though